frontend/src/charts/chartOperations.ts

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { LabelStyle } from 'charts/Charts.types';
+
+export const getFontWeight = (style: LabelStyle | null): string => {
+  if (style?.bold != null) {
+    return style.bold ? 'bold' : 'normal';
+  }
+  return 'normal';
+};
+
+export const getFontSize = (style: LabelStyle | null): number => {
+  return style?.fontSize ?? 10;
```

Comment:
14 is the default fontsize, why use 10?

Reply:
Ok for 14 (10 is the D3 default value). More generally, we will need to figure it out how to deal with the displayed chart size.